website/docs/pushdown.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 6
+hide_table_of_contents: true
+---
+
+# Query pushdown
+
+**Query pushdown** is the method of optimizing a query by reducing the amount of data read and processed.
+It saves memory space, network bandwidth, and reduces the query execution time by not prefetching
+unnecessary data to the database management system.
+
+Initially FerretDB retrieved all data related to queried collection, and applies filters on its own, making
+it possible to implement complex logic safely and quickly.
+To make this process more efficient, we minimize the amount of incoming data, by applying proper SQL filters.
+
+:::info
+You can learn more about query pushdown in our [blog post](https://blog.ferretdb.io/ferretdb-fetches-data-query-pushdown/).
+:::
+
+## Supported types and operators
+
+The following table shows all operators and types that FerretDB pushdowns on PostgreSQL backend.
+If filter uses type and operator, that's marked as pushdown-supported on this list,
+FerretDB will prefetch less data, resulting with more performent query.
+
+If your application requires better performance for specific operation,
+feel free to share this with us on [FerretDB Slack](https://join.slack.com/t/ferretdb/shared_invite/zt-zqe9hj8g-ZcMG3~5Cs5u9uuOPnZB8~A)!
+
+:::tip
+As query pushdown allows developers to implement query optimizations separately from the features,
+the table will be updated frequently.
+:::
+
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable MD001 MD033 MD051 -->
+|            | Object   | Array   | Double                 | String   | Binary   | ObjectID   | Boolean   | Date   | Null   | Regex   | Integer   | Timestamp   | Long                   |
```

Comment:
I remember, Alexey had some questions about this table formatting (it was too wide), but I guess it's solved?

Reply:
Yes, we removed the sidebar, you can see final results [here](https://c1f7bf7b.ferretdb-docs-dev.pages.dev/pushdown/)!